sdk/ai/azure-ai-generative/azure/ai/generative/synthetic/simulator/_model_tools/models.py

```diff
@@ -723,20 +774,21 @@ def format_request_data(self, prompt: str, **request_params):
         request_data = {
             "input_data": {
                 "input_string": [prompt],
-                "parameters": {"temperature": self.temperature, "max_gen_len": self.max_tokens}
+                "parameters": {"temperature": self.temperature, "max_gen_len": self.max_tokens},
             }
         }
 
         request_data.update(request_params)
         return request_data
 
+    # pylint: disable=arguments-differ
     def _parse_response(self, response_data: dict, request_data: dict) -> dict:  # type: ignore[override]
-        prompt = request_data['input_data']['input_string'][0]
+        prompt = request_data["input_data"]["input_string"][0]
 
         # remove prompt text from each response as llama model returns prompt + completion instead of only completion
         # remove any text after the stop tokens, since llama doesn't support stop token
-        for idx, response in enumerate(response_data["samples"]):
-            response_data["samples"][idx] = response_data["samples"][idx].replace(prompt, '').strip()
+        for idx, _ in enumerate(response_data["samples"]):
+            response_data["samples"][idx] = response_data["samples"][idx].replace(prompt, "").strip()
```

Comment:
nit:

We could used the second value from enumerate

```python
for idx, response in enumerate(response_date["samples"]):
     response_data["samples][idx] = response.replace(prompt, "").strip()
```

Reply:
fixed